tests/controllers/qos_controller_test.cpp

```diff
@@ -1,17 +1,21 @@
-#include <gtest/gtest.h>
+#include <list>
 
-#include <mesos/resources.hpp>
+#include "gtest/gtest.h"
 
-#include <mesos/slave/oversubscription.pb.h>  // ONLY USEFUL AFTER RUNNING PROTOC
-#include <mesos/slave/qos_controller.hpp>
+#include "mesos/resources.hpp"
 
-#include <stout/gtest.hpp>
+// ONLY USEFUL AFTER RUNNING PROTOC
```

Comment:
See above comments - we do follow Mesos style as much as Cpp Linter allows us (: